Project.toml

```
@@ -12,11 +12,11 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-ChainRulesCore = "0.9.21"
-ChainRulesTestUtils = "0.5.5"
+ChainRulesCore = "0.9.25"
+ChainRulesTestUtils = "0.5, 0.6"
```

Currently Julia v1.5 passes but only with v0.6.1
```suggestion
ChainRulesTestUtils = "0.5, 0.6.1"
```